Microsoft.Packaging.Tools/tasks/Trimming/FileNode.cs

```diff
@@ -67,27 +92,44 @@ public void PopulateDependencies(Dictionary<string, FileNode> allFiles, bool pre
                     if (peReader.HasMetadata)
                     {
                         var reader = peReader.GetMetadataReader();
+
+                        var includeDependencies = true;
+
+                        // map of facade handles to enable quickly getting to FileNode without repeatedly looking up by name
+                        var facadeHandles = new Dictionary<AssemblyReferenceHandle, FileNode>();
+
+                        if (IsFullFacade(reader))
```

Comment:
The problem with partial facades is that there is no good way to safely eliminate all dependencies of the assembly because its not a simple task to understand what references are there for the types in the assembly vs the type forwards.  I have to root the facade since the runtime will need to load it.  If I can't eliminate the dependencies of the facade DLL then there is no point in following the typeforwards.  Let's leave that level of analysis for the linker.